docs/docsite/rst/scenario_guides/guide_meraki.rst

```diff
@@ -0,0 +1,128 @@
+.. _meraki_guide:
+
+Cisco Meraki Guide
+==================
```

Comment:
Top header should use ****** above and below; next level down should use ====== only underneath - see #35520

Reply:
@acozine I'll be submitting a new PR for this in a bit. Please review and make sure it's what you expected.